### PR TITLE
Fix for compiler warnings with -Wextra

### PR DESCRIPTION
**Pull request by drzony**

### CODE REVIEW

File: src/internal/Esp32_i2s.c [hunk 1]
```diff
@@ -156,8 +156,12 @@ bool i2sInitDmaItems(uint8_t bus_num) {
         item->free_ptr = NULL;
         item->buf = NULL;
     }
-    itemPrev->eof = 1;
-    item->eof = 1;
+    if (itemPrev) {
```
Comment (Makuna) [inline src/internal/Esp32_i2s.c:159]: Is there some other way to suppress the warning than by adding code here?

There is no possible way to get to this point with these being NULL.

**Review by Makuna — CHANGES_REQUESTED** [review]

File: src/internal/Esp32_i2s.c (hunk 1 above)
Comment (drzony) [reply src/internal/Esp32_i2s.c]: It's not needed to remove the warning, just wanted to be thorough.

**Comment by drzony** [comment]
I've changed things according to request, any chance to merge this?

**Comment by Makuna** [comment]
> I've changed things according to request, any chance to merge this?

I can't access the review nor the changes that were made to fix it.  It just shows as a pending Review (waiting on you to make changes and check the request line items off).

**Pull request merged**